client/QuickStart/API_Util.cs

```
@@ -0,0 +1,410 @@
+    /// </summary>
+    public static class Util
+    {
+        private const string MgmtPrefix = ".";
+        /// <summary>
+        /// Loads JSON configuration file, and sets the metadata in place. 
+        /// </summary>
+        /// <param name="configFilePath"> Configuration file path.</param>
+        /// <returns>ConfigJson object, allowing access to the metadata fields.</returns>
+        public static ConfigJson LoadConfigs(string configFilePath)
+        {
+            try
+            {
+                var json = File.ReadAllText(configFilePath);
+                var config = JsonConvert.DeserializeObject<ConfigJson>(json);
+                var missing = new[]
+                {
+                    (name: nameof(config.DatabaseName), value: config.DatabaseName),
+                    (name: nameof(config.TableName), value: config.TableName),
+                    (name: nameof(config.TableSchema), value: config.TableSchema),
+                    (name: nameof(config.KustoUri), value: config.KustoUri),
+                    (name: nameof(config.IngestUri), value: config.IngestUri),
+                    (name: nameof(config.AuthenticationMode), value: config.AuthenticationMode)
+                }.Where(item => string.IsNullOrWhiteSpace(item.value)).ToArray();
+
+                if (missing.Any())
+                    ErrorHandler($"File '{configFilePath}' is missing required fields: {string.Join(", ", missing.Select(item => item.name))}");
+
+                if (config.Data is null || !config.Data.Any() || config.Data[0].Count == 0)
+                    ErrorHandler($"Required field Data in '{configFilePath}' is either missing, empty or misfilled");
+                return config;
+            }
+            catch (Exception ex)
+            {
+                ErrorHandler($"Couldn't read config file: '{configFilePath}'", ex);
+            }
+
+            return null;
```

Thinking about it, I don't like returning null in this state. If we had a bug and it reached here, we will get a subtle NRE later.
Maybe instead do `throw new InvalidOperationException("Unreachable code");`?